### PR TITLE
.amlignore example of usage 

### PR DESCRIPTION
**Pull request by sbaidachni**

### CODE REVIEW

**Comment by dtzar** [comment]
This makes sense, however at the directory you put the file there are no .yml files underneath it.  If you put it at the root it would make sense IF someone specified the root source directory to be transferred to remote compute.  I'd also link to a doc/site where it talks about .amlignore.

**Comment by sbaidachni** [comment]
@dtzar There are at least 4 yml files in scoring subfolder:) The root folder doesn't make any sense, because you submit diabetes folder to aml compute. So, everything that is outside the diabetes folder should be ignored by the PythonScriptStep including amlignore:)

File: diabetes_regression/.amlignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# We use yml files to configure deployment,
+# but we are not deploying them to compute
```
Comment (dtzar) [inline diabetes_regression/.amlignore]: Please put a better description of what this does here IF there is no link to a doc with what this does.

**Comment by dtzar** [comment]
Ok, no problem on the location & .yml file specification.  Would be nice to know how people can ignore a directory (even if it isn't present under there) IF there is no public document explaining it.  So... with a public link you should be able to just comment with the link for more info.  No link, then you need better description + folder exclusion example.

**Comment by sbaidachni** [comment]
Will it be OK if I just add a note that the format of the file is identical to .gitignore?

**Comment by dtzar** [comment]
Yes but I remember @xinyi-joffre stating something like there are some quirks to .amlignore where it doesn't behave exactly like this?  Is there ANY documentation referencing .amlignore or you just need tribal knowledge?

**Comment by xinyi-joffre** [comment]
The documentation about .amlignore is usually in notes in the docs. For example, [here](https://docs.microsoft.com/en-us/azure/machine-learning/concept-azure-machine-learning-architecture#snapshots). It may also be worthwhile to leave a comment about how this file can be used to reduce snapshot size if you get errors about too many files in snapshot or if it surpasses the max snapshot size.

The .amlignore should theoretically behave the same way as .gitignore. When we initially used it, the underlying implementation was broken for the exclude all pattern where you start with * and then add files back into snapshot. Also, we had to make sure that we included the .amlignore itself in the snapshot. We reported it to product team, and according to this [post](https://stackoverflow.com/questions/58345935/the-amlignore-file-doesnt-reduce-the-size-of-snapshot), the exclude all bug was fixed in 1.0.74 of azureml-sdk, but I haven't tested to make sure it works.

Azure ML has their own logic for building a tree of the hash paths so there is still a possibility of it unintentionally behaving slightly differently than .gitignore.

If you are only ignoring files directly (such as *.yml), it should work well.

**Comment by dtzar** [comment]
Thanks @xinyi-joffre!  I think the link you provided is sufficient for the comment :)

**Pull request merged**